lib/ids_please/grabbers/base.rb

```diff
@@ -3,30 +3,38 @@
 class IdsPlease
   module Grabbers
     class Base
-
       def self.interact(links)
-        links.map { |l| self.new(l).grab_link }
+        links.map do |l|
+          new(l).grab_link
+        end
```

Comment:
Style preferences again with `{...}` instead of `do...end` with maps and using `self` with `new`